.github/workflows/base_node_binaries.json

```diff
@@ -54,6 +53,7 @@
     "cross": false,
     "target_cpu": "generic",
     "features": "safe",
+    "target_bins": "--bin tari_base_node --bin tari_console_wallet --bin tari_merge_mining_proxy --bin tari_miner --bin tari_validator_node",
```

Comment:
Looks like the VN is still in this one

Reply:
Disabled build target - windows-arm64 (not yet supported) but should remove the reference.